src/index.js

```diff
@@ -178,14 +210,59 @@ const moduleToApp = (ddoc, module) =>
     digest: ddoc._attachments[module].digest,
   });
 
+const deployDdoc = stagedDdoc => {
+  info(`Deploy staged ddoc ${stagedDdoc._id} to ${DDOC}`);
```

Comment:
Yeah… I should pick either using generic names everywhere (but hard-coding at the top) or hard-coded names everywhere and dealing with generics when / if we get there.

I'll change it to STAGED_DDOC, I'd rather not set expectations the wrong way.